pallets/automation-time/src/migrations/upgrade_xcmp_task_struct.rs

```diff
@@ -51,7 +51,6 @@ pub enum OldAction<T: Config> {
 		delegator: T::AccountId,
 		collator: T::AccountId,
 		account_minimum: BalanceOf<T>,
-		frequency: Seconds,
```

Comment:
Would this change the interface(parameters) of the existing `scheduleAutoCompoundDelegatedStakeTask` call on Turing Network?

In other words, what is the parameter list of `scheduleAutoCompoundDelegatedStakeTask` after this migration?

Reply:
This line is not needed because the Schedule data is already migrated to Fixed and Recurring formats on Turing Network